services/libs/data-access-layer/src/old/apps/data_sink_worker/repo/organization.repo.ts

```diff
@@ -130,21 +51,20 @@ export class OrganizationRepository extends RepositoryBase<OrganizationRepositor
               select oi."organizationId"
               from "organizationIdentities" oi
               where 
-                    oi.platform = $(platform)
-                    and oi.name ilike $(name)
+                    oi."tenantId" = $(tenantId)
+                    and oi.platform = $(platform)
+                    and oi.value ilike $(value)
```

Comment:
let's use `lower(oi.value) = lower($(value))` match here instead (like we do in members) it's performing much better with an index on lower(value)